client/my-sites/themes/single-site.jsx

```diff
@@ -9,74 +9,18 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import Main from 'components/main';
-import CurrentTheme from 'my-sites/themes/current-theme';
-import SidebarNavigation from 'my-sites/sidebar-navigation';
-import ThanksModal from 'my-sites/themes/thanks-modal';
-import config from 'config';
-import EmptyContent from 'components/empty-content';
-import JetpackUpgradeMessage from './jetpack-upgrade-message';
-import JetpackManageDisabledMessage from './jetpack-manage-disabled-message';
-import { connectOptions } from './theme-options';
+import SingleSiteThemeShowcase from './single-site-wpcom';
+import SingleSiteThemeShowcaseJetpack from './single-site-jetpack';
```

Comment:
Yeah I wasn't convinced because "Wpcom" seemed a weird capitalization of what's already an acronym, but ok. :)